src/hooks/useFirebase.js

```diff
@@ -0,0 +1,10 @@
+import { useEffect, useState } from 'react';
+import getFirebase from '../config/firebase';
+
+export default function useFirebase() {
+  const [instance, setInstance] = useState(null);
+  useEffect(() => {
+    setInstance(getFirebase());
+  }, []);
+  return instance;
```

Comment:
Why do we need to use `useEffect` and `useState` here?

Can't we do the following?
```
  const [instance] = useState(getFirebase());
  return instance;
```

Reply:
Yes, we can use do that also... I will take care of it.